java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -690,6 +690,15 @@ public static ColumnVector concatenate(ColumnVector... columns) {
     }
   }
 
+  private static native long normalizeNANsAndZeros(long viewHandle) throws CudfException;
+
+  public ColumnVector normalizeNANsAndZeros() {
```

Comment:
Please add in some javadocs to explain what this is doing.

Reply:
Righto. Will do shortly.

Reply:
Updated docs.